test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -295,6 +293,28 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 			}
 		})
 
+		It("should validate that every linux node has the right sshd config", func() {
+			if eng.ExpandedDefinition.Properties.IsUbuntuDistroForAllNodes() {
+				kubeConfig, err := GetConfig()
+				Expect(err).NotTo(HaveOccurred())
+				master := fmt.Sprintf("azureuser@%s", kubeConfig.GetServerName())
+				nodeList, err := node.Get()
+				Expect(err).NotTo(HaveOccurred())
+				sshdConfigs := []string{"ClientAliveInterval 120", "ClientAliveCountMax 3", "UsePrivilegeSeparation yes", "KeyRegenerationInterval 3600", "ServerKeyBits 1024", "SyslogFacility AUTH", "LogLevel INFO", "LoginGraceTime 60", "PermitRootLogin no", "PermitUserEnvironment no", "StrictModes yes", "RSAAuthentication yes", "PubkeyAuthentication yes", "IgnoreRhosts yes", "RhostsRSAAuthentication no", "HostbasedAuthentication no", "X11Forwarding no", "MaxAuthTries 4", "Banner /etc/issue.net", "AcceptEnv LANG", "AcceptEnv LC_*", "Subsystem sftp /usr/lib/openssh/sftp-server", "UsePAM yes", "UseDNS no", "GSSAPIAuthentication no"}
+				for _, node := range nodeList.Nodes {
+					for _, config := range sshdConfigs {
```

Comment:
It may be easy to deliver a single script and then enumerate through configs in that script to avoid ssh'ing a bunch of times. Since that pattern landed after you began this PR, you can deliver that as a follow-on PR as well if we have the actual sshd config validation we need in this changeset.